src/System.Web.HttpUtility/src/System/Web/HttpUtility.cs

```diff
@@ -256,7 +256,7 @@ public static byte[] UrlEncodeToBytes(byte[] bytes, int offset, int count)
          )]
         public static string UrlEncodeUnicode(string str)
         {
-            return HttpEncoder.UrlEncodeUnicode(str, ignoreAscii: false);
+            return HttpEncoder.UrlEncodeUnicode(str);
```

Comment:
nit: looks like this should have been in the previous commit, but not important

Reply:
That's an annoying oversight on my part, but it'll amount to nothing after squashing, anyway.